diff -u ocs.inventory.client.cron ocs.inventory.client.cron
--- ocs.inventory.client.cron	2006-07-20 15:13:09.000000000 -0500
+++ ocs.inventory.client.cron	2011-12-08 11:13:41.039012018 -0600
@@ -4,6 +4,7 @@
 #
 #  Written by Troy Dawson <dawson@fnal.gov>
 #	Original - May 24, 2006  by Troy Dawson <dawson@fnal.gov>
+#       Updated - December 8, 2011 by Connie Sieh <csieh@fnal.gov>
 #
 #Change DEBUG to 0 to turn on debugging
 DEBUG=1
@@ -56,7 +57,7 @@
 	# Loop through FERMINET and see if its in the ifconfig
 	for TESTNET in $FERMINET
 	do
-		ifconfig | grep -q $TESTNET
+		ip addr | grep -q $TESTNET
 		if [ $? -eq 0 ] ; then
 			if [ $DEBUG -eq 0 ] ; then
 				echo "  This machine is on the Fermilab Network"
